packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -47,23 +72,33 @@ class Product extends Component {
     }
 
     get options() {
-        const { classes, item } = this.props;
-
-        return item.options && item.options.length > 0 ? (
-            <dl className={this.props.classes.options}>
-                {item.options.map(({ name, value }) => (
-                    <Fragment key={name}>
-                        <dt className={classes.optionName}>{name}</dt>
-                        <dd className={classes.optionValue}>{value}</dd>
-                    </Fragment>
-                ))}
-            </dl>
-        ) : null;
+        const { classes, item, totalsItems } = this.props;
+        const { item_id } = item;
+        const regex = /[\[\]']+/g; // regex to to remove [] , use it to check if options are really empty
+        return totalsItems
+            .filter(totalsItem => totalsItem.item_id === item_id)
+            .map(totalsItem => {
+                const { options } = totalsItem;
+                return options && options.replace(regex, '').length > 0 ? (
+                    <dl className={this.props.classes.options} key={item_id}>
+                        {JSON.parse(options).map(option => (
+                            <Fragment key={`${item_id}_${option.label}`}>
+                                <dt className={classes.optionLabel}>
+                                    {option.label}:&nbsp;
+                                </dt>
+                                <dd className={classes.optionValue}>
+                                    {option.value}
+                                </dd>
+                            </Fragment>
+                        ))}
+                    </dl>
+                ) : null;
+            });
     }
 
     styleImage(image) {
         return {
-            height: imageHeight,
+            minHeight: imageHeight, // min-height instead of height so image will always align with grid bottom
```

Comment:
I don't have a problem with setting `min-height` but do we want to still set `height` also?

Also it's a little bit of a red flag that we're relying on heights to properly align things. We should use [align-items](https://css-tricks.com/snippets/css/complete-guide-grid/#prop-align-items) for alignment.

Reply:
This setup is mainly caused by the fact that we use grid here and the images are background images. So we get the following:

- Setting an explicit height to the image will restrict the image, so it won't fill the complete grid area. Screenprint: explicit height:
![magento-pwa-studio-pull-582-image-height-instead-of-min-height](https://user-images.githubusercontent.com/1906257/49529448-5181d680-f8b6-11e8-94ee-f31ebb686b21.png)
- Setting no height, won't always render the complete image (for example in case of simple products). screenprint: no height 
![magento-pwa-studio-pull-582-image-no-height](https://user-images.githubusercontent.com/1906257/49529484-62cae300-f8b6-11e8-8f8c-96099a768ea6.png)
- so min-height seems the best fit here.

Reply:
Thanks for the explanation 👍